cmd/common-main.go

```diff
@@ -177,6 +177,10 @@ func minioConfigToConsoleFeatures() {
 			os.Setenv("CONSOLE_LOG_QUERY_AUTH_TOKEN", value)
 		}
 	}
+	// pass the console subpath configuration
+	if value := env.Get("MINIO_CONSOLE_SUBPATH", ""); value != "" {
+		os.Setenv("CONSOLE_SUBPATH", value)
+	}
```

Comment:
as discussed we should re-use the `MINIO_BROWSER_REDIRECT_URL` with `subpath` to ensure that subpath is honored @dvaldivia instead of a new ENV